rust/src/dns/log.rs

```diff
@@ -86,6 +86,7 @@ pub const LOG_URI        : u64 = BIT_U64!(59);
 
 pub const LOG_FORMAT_GROUPED  : u64 = BIT_U64!(60);
 pub const LOG_FORMAT_DETAILED : u64 = BIT_U64!(61);
+pub const LOG_HTTPS      : u64 = BIT_U64!(62);
```

Comment:
Can we remove these LOG_HTTPS entries? I see it being set, but never read suggesting its not needed at this time.

Reply:
Ah, maybe I'm wrong. IMO, an example of too much configuration for a protocol, and I've forgotten how it works.

Reply:
It is getting used on line 255